### ISSUE TITLE
No test result

### ISSUE THREAD

**Issue by huangchaoxing** [tegg89/SRCNN-Tensorflow#20]
when i ran the sentence, python main.py --is_train False --stride 21, I COULD only find one image in the file sample, i am not sure where the SR result is ...

**Comment by PrathamSoni** [comment]
Test_image.png is the SR result. You can find the original in your test folder

**Comment by huangchaoxing** [comment]
@PrathamSoni 
when i ran the sentence python main.py --is_train False --stride 21, i find that the output shrink from 256*256 to 231*231, any idea why the heck this happen ?

**Comment by PrathamSoni** [comment]
It deletes some of the padding area.

**Comment by woshizhangcheng** [comment]
请问你这个问题解决了吗？

**Comment by niuniu111** [comment]
I tested nothing and I didn't know what was going on.

**Comment by Marshall-yao** [comment]
Hello,when i ran the sentence python main.py --is_train False --stride 21, i find that test could not run.The result is shown below:
2019-02-19 15:34:31.849830: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-02-19 15:34:32.215031: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Found device 0 with properties:
name: GeForce GTX TITAN X major: 5 minor: 2 memoryClockRate(GHz): 1.076
pciBusID: 0000:82:00.0
totalMemory: 11.93GiB freeMemory: 1.51GiB
2019-02-19 15:34:32.215092: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX TITAN X, pci bus id: 0000:82:00.0, compute capability: 5.2)
WARNING:tensorflow:From /home/songhuihui/anaconda3/envs/tf_env/lib/python3.5/site-packages/tensorflow/python/util/tf_should_use.py:107: initialize_all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use `tf.global_variables_initializer` instead.
 [*] Reading checkpoints...
 [*] Load SUCCESS
Testing...
(tf_env) songhuihui@node2:~/usrs/yaolu/tf/reproSRCNN/tegg$

how could i  run test code? thanks a lot

**Comment by vinovsmamba** [comment]
why when I ran the sentence python main.py --is_train False --stride 21, it still goes train process?

(tensorflow) F:\F_coding\other\1_SRCNN-Tensorflow-master>python main.py --is_train False --stride 21
C:\Users\xxxx\AppData\Roaming\Python\Python36\site-packages\h5py\__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
{'batch_size': <absl.flags._flag.Flag object at 0x000001F92171B400>,
 'c_dim': <absl.flags._flag.Flag object at 0x000001F92B640B38>,
 'checkpoint_dir': <absl.flags._flag.Flag object at 0x000001F92B640D30>,
 'epoch': <absl.flags._flag.Flag object at 0x000001F92172F4E0>,
 'h': <tensorflow.python.platform.app._HelpFlag object at 0x000001F92B640EB8>,
 'help': <tensorflow.python.platform.app._HelpFlag object at 0x000001F92B640EB8>,
 'helpfull': <tensorflow.python.platform.app._HelpfullFlag object at 0x000001F92B640F28>,
 'helpshort': <tensorflow.python.platform.app._HelpshortFlag object at 0x000001F92B640F98>,
 'image_size': <absl.flags._flag.Flag object at 0x000001F923CEAF28>,
 'is_train': <absl.flags._flag.BooleanFlag object at 0x000001F92B640DD8>,
 'label_size': <absl.flags._flag.Flag object at 0x000001F925DAB780>,
 'learning_rate': <absl.flags._flag.Flag object at 0x000001F92B3E9128>,
 'sample_dir': <absl.flags._flag.Flag object at 0x000001F92B640DA0>,
 'scale': <absl.flags._flag.Flag object at 0x000001F92B640BE0>,
 'stride': <absl.flags._flag.Flag object at 0x000001F92B640C88>}
2019-04-12 16:47:53.584329: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2019-04-12 16:47:53.814883: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties:
name: GeForce GTX 1060 major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:01:00.0
totalMemory: 3.00GiB freeMemory: 2.42GiB
2019-04-12 16:47:53.836075: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-04-12 16:47:54.455790: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-04-12 16:47:54.465749: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0
2019-04-12 16:47:54.472550: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N
2019-04-12 16:47:54.480301: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 2109 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1060, pci bus id: 0000:01:00.0, compute capability: 6.1)
WARNING:tensorflow:From D:\scholarship\anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\framework\op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From D:\scholarship\anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\util\tf_should_use.py:193: initialize_all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use `tf.global_variables_initializer` instead.
 [*] Reading checkpoints...
WARNING:tensorflow:From D:\scholarship\anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\training\saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
 [*] Load SUCCESS
Training...
Epoch: [ 1], step: [10], time: [1.2915], loss: [0.00145336]
Epoch: [ 1], step: [20], time: [1.3823], loss: [0.00140325]
Epoch: [ 1], step: [30], time: [1.4721], loss: [0.00168100]
Epoch: [ 1], step: [40], time: [1.5599], loss: [0.00023895]
………………


I ran it on Anaconda Prompt in tensorflow environment. It trains well but just can't test. 
I have to rewrite the code to change the number and string in flags.DEFINE.
